docs/sources/binding/android.md

```diff
@@ -1,29 +1,68 @@
 # Android
 
-* **Android:** - change your base class to one of the Reactive Activity /
-  Fragment classes (i.e. ReactiveActivity<T>), *or* implement
-  `INotifyPropertyChanged` on your View and ensure that ViewModel signals
-  changes.
+Change your base class to one of the Reactive Activity / Fragment classes 
+(i.e. ReactiveActivity<T>), *or* implement `IViewFor` on your View 
+and ensure that your ViewModel signals changes.
+  
+```csharp
+public class TheViewModel : ReactiveObject
+{
+    private string theText;
+    
+    public string TheText
+    {
+        get { return this.theText; }
+        set { this.RaiseAndSetIfChanged(ref this.theText, value); }
+    }
+}
+```
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  android:orientation="vertical"
+  android:layout_width="fill_parent"
+  android:layout_height="fill_parent">
+  <EditText
+    android:id="@+id/TheEditText"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" />
+  <TextView
+    android:id="@+id/TheTextView"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" />
+</LinearLayout>
+```
 
 ```csharp
 [Activity (Label = "RxUISample-Android", MainLauncher = true)]
-public class TestActivity : ReactiveActivity, IViewFor<TestViewModel>
-{
-    protected override async void OnCreate(Bundle bundle)
+public class TestActivity : ReactiveActivity, 
+  IViewFor<TheViewModel> // Or use ReactiveActivity<TheViewModel>, which implements IViewFor<TheViewModel>
```

Comment:
Yeah, I wasn't sure about that either, in the end I implemented `IViewFor<>` explicitly, so the reader can see that there is a `ViewModel` property, like in the Android sample that I created